### PR TITLE
Get latest changes 11/21

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/office/office.activeview.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.activeview.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.binding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.binding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.bindings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.bindings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.bindingtype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.bindingtype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.coerciontype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.coerciontype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlnode.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.customxmlnode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlnodetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.customxmlnodetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlpart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.customxmlpart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlparts.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.customxmlparts.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.customxmlprefixmappings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.customxmlprefixmappings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.document.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.document.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.documentmode.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.documentmode.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.file.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.file.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.filetype.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.filetype.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.matrixbinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.matrixbinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.nodedeletedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.nodedeletedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.nodeinsertedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.nodeinsertedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.nodereplacedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.nodereplacedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.settings.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.settings.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.table.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.table.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.tablebinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.tablebinding.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.textbinding.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.textbinding.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/api-extractor-inputs-office/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/office.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/json/office.api.json) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C21%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201811211639439372-247%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=JVxFkCyQz3Xi3XBQ2Xtsg8UtmaGxlP2ng2iFr%2FibWcU%3D&st=2018-11-21T16%3A40%3A10Z&se=2018-12-22T16%3A45%3A10Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docs-ref-autogen/office/office.document.yml [hunk 1]
```diff
@@ -3552,10 +3557,11 @@ items:
 
 
       *Supported hosts, by platform* <table> <tr><th> </th><th> Office for Windows desktop </th><th> Office Online (in
-      browser) </th><th> Office for iPad </th></tr> <tr><td><strong> Access </strong></td><td> </td><td> Y </td><td>
-      </td></tr> <tr><td><strong> Excel </strong></td><td> Y </td><td> Y </td><td> Y </td></tr> <tr><td><strong>
-      PowerPoint </strong></td><td> Y </td><td> Y </td><td> Y </td></tr> <tr><td><strong> Word </strong></td><td> Y
-      </td><td> Y </td><td> Y </td></tr> </table>
+      browser) </th><th> Office for iPad </th><th> Office for Mac </th></tr> <tr><td><strong> Access </strong></td><td>
```
Comment (kbrandl) [inline docs/docs-ref-autogen/office/office.document.yml]: Should the "Supported hosts, by platform" table within the **url** property section also contain a column for "Office for Mac" ?

File: docs/docs-ref-autogen/office/office.settings.yml [hunk 2]
```diff
@@ -133,8 +133,8 @@ items:
 
 
             *Supported hosts, by platform* <table> <tr><th> </th><th> Office for Windows desktop </th><th> Office Online
-            (in browser) </th><th> Office for iPad </th></tr> <tr><td><strong> Excel </strong></td><td> </td><td> Y
-            </td><td> </td></tr> </table>
+            (in browser) </th><th> Office for iPad </th></tr> <tr><td><strong> Access </strong></td><td> </td><td> Y
+            </td><td> </td></tr> <tr><td><strong> Excel </strong></td><td> </td><td> Y </td><td> </td></tr> </table>
```
Comment (kbrandl) [inline docs/docs-ref-autogen/office/office.settings.yml]: Should this table also contain a column for "Office for Mac" ?

File: docs/docs-ref-autogen/office/office.document.yml (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/office/office.document.yml]: I only have info for methods from the SOT and not properties so I couldn't verify that. I was able to make a best guess for some of the properties (e.g. the platform for that host supports the entire object or the requirement set and so by extension supports any and all methods and properties) but I couldn't make that determination for this property and others like it.

In DefinitelyTyped, I can certainly add the column but I don't have the details needed to fill it for each host.

Comment (kbrandl) [reply docs/docs-ref-autogen/office/office.document.yml]: Thanks for the explanation. If we don't have the information necessary to confidently populate the additional column, probably best to omit it for now.

**Pull request merged**